tobac/feature_detection.py

```diff
@@ -1350,20 +1393,20 @@ def feature_detection_multithreshold(
 
 
 def filter_min_distance(
-    features,
-    dxy=None,
-    dz=None,
-    min_distance=None,
-    x_coordinate_name=None,
-    y_coordinate_name=None,
-    z_coordinate_name=None,
-    target="maximum",
-    PBC_flag="none",
-    min_h1=0,
-    max_h1=0,
-    min_h2=0,
-    max_h2=0,
-):
+    features: pd.DataFrame,
+    dxy: float = None,
+    dz: float = None,
+    min_distance: float = None,
+    x_coordinate_name: str = None,
+    y_coordinate_name: str = None,
+    z_coordinate_name: str = None,
+    target: Literal["maximum", "minimum"] = "maximum",
+    PBC_flag: Literal["none", "hdim_1", "hdim_2", "both"] = "none",
+    min_h1: int = 0,
+    max_h1: int = 0,
+    min_h2: int = 0,
+    max_h2: int = 0,
```

Comment:
are these variables `int` or can they be `float`? I've been using float values with PBCs for tracking, but I'm not sure if it's the same for feature detection

Reply:
These variables cannot be `float`; they should be array indices.

Reply:
Can we specify that these are array indices in the docstrings? I interpreted it as the actual dimension values/coordinates and I am surprised this did not change my results for the MCSMIP tracking now when I corrected it.

Reply:
Also, I realized that these are not required by the user when running the feature detection with PBC, while the tracking with PBC requires the user to define these (but that should be another issue), just as the docstrings.

Reply:
Sorry for posting this here. I know this should be a separate issue, but I wanted to clarify first if I interpret it right.